hamilton/io/utils.py

```diff
@@ -17,3 +17,7 @@ def get_file_metadata(path: str) -> Dict[str, Any]:
         "last_modified": os.path.getmtime(path),
         "timestamp": datetime.now().utcnow().timestamp(),
     }
+
+
+def get_sql_metadata() -> Dict[str, Any]:
+    pass
```

Comment:
yep we need to think about what to put here?

Maybe we take the original DF and do "len" on it, and pull a few other things, and just the table details? thoughts?

Reply:
yep, makes sense to me. added in commit [07cc882](https://github.com/DAGWorks-Inc/hamilton/pull/353/commits/07cc88274db672f4591240f1066a6dd3426f6e58).